warehouse/core/src/main/java/datawave/mr/bulk/BulkInputFormat.java

```diff
@@ -1289,8 +1291,8 @@ public AccumuloIteratorOption(String iteratorOption) {
             StringTokenizer tokenizer = new StringTokenizer(iteratorOption, FIELD_SEP);
             this.iteratorName = tokenizer.nextToken();
             try {
-                this.key = URLDecoder.decode(tokenizer.nextToken(), "UTF-8");
-                this.value = URLDecoder.decode(tokenizer.nextToken(), "UTF-8");
+                this.key = URLDecoder.decode(tokenizer.nextToken(), UTF8);
```

Comment:
Why not `Charset.UTF_8`?

Reply:
This is just replacing a String of "UTF-8", not a character set or anything fancy.